docs/quickstart.md

```diff
@@ -188,7 +188,7 @@ with Tensor.train():
     out = net(batch)
 
     # compute loss
-    loss = sparse_categorical_crossentropy(out, labels)
+    loss = out.sparse_categorical_crossentropy(labels)
```

Comment:
It's defined on line 138. This was to show the implementation that is defined above.

Reply:
Ah, got it. I definitely skipped that codeblock when I started to write the training code portion.